components/senza.rst

```diff
@@ -514,6 +520,84 @@ to $0.134 per hour.
           InstanceType: c4.large
           SpotPrice: 0.134
 
+Senza also supports `Spotinst's`_ `Elastigroup`_. For details how that works with
+senza read the section `Senza::Elastigroup`_
+
+Senza::Elastigroup
+~~~~~~~~~~~~~~~~~~
+
+The **Elastrigoup** component type creates an Elastigroup. It's the equivalent of
+an Auto Scaling Group, but managed externally by a third party - Spotinst.
+
+Quote from the vendor: "Spotinst Elastigroup predicts EC2 Spot behavior, capacity
+trends, pricing, and interruptions rate. Whenever there’s a risk of interruption,
+Elastigroup acts accordingly to balance capacity up to 15 minutes ahead of time,
+ensuring 100% availability."
+
+.. code-block:: yaml
+
+    SenzaComponents:
+      - AppServer:
+          Type: Senza::Elastigroup
+          InstanceType: "c5.large"
+          SpotAlternatives:
+              - "m5.large"
+              - "c5.xlarge"
+          SecurityGroups: app-hello-world
+          IamRoles:
+            - app-hello-world
+          ElasticLoadBalancerV2: AppLoadBalancer
+          TaupageConfig:
+            runtime: Docker
+            source: pierone.example.org/foobar/myapp:1.0
+            ports:
+              8080: 8080
+            environment:
+              FOO: bar
+
+This component accepts ALL of the properties of the `Senza::TaupageAutoScalingGroup`_
+component. They are mapped to the corresponding attributes of the Elastigroup.
+This includes the ``AutoScaling`` properties.
+
+It adds the following additional properties:
+
+``SpotAlternatives``
+    The EC2 instance types that should be used as Spot instead of the On-Demand
+    ``InstanceType``. The selection of which one is effectivelly used is controlled
+    by the `Elastigroup cluster orientation`_. The default setting is "Balanced".
+
+``Elastigroup``
+    The raw specification of the Elastigroup, according to the `Elastigroup API`_.
+    The content of this property is set as the content of the ``group`` attribute
+    of the API.
```

Comment:
Maybe nitpicking: Could we add here "Refer to the vendor documentation for the definition of the Elastigroup properties."
I was looking at the example back and forth trying to find `Elastigroup` there.